indexing-hadoop/src/test/resources/druid.test.data.tsv

```diff
@@ -0,0 +1,82 @@
+2011-04-10T00:00:00.000Z	spot	automotive	preferred	apreferred	113.221448
```

Comment:
First I tested with the existing druid.sample.tsv file but then I thought the sample data should have some duplicate rows for a shard so that HyperLogLog unique count feature can be tested. If the files are kept separate, more interesting tests can be added in future. However, I am fine with any approach.